providers/ms365/resources/ms365.lr

```diff
@@ -72,18 +72,36 @@ microsoft.tenant @defaults("name") {
   subscriptions() []dict
   // Company-wide settings for apps and services.
   settings() microsoft.tenant.settings
+  // Company-wide settings for Microsoft Forms
+  formsSettings() microsoft.tenant.formsSettings
 }
 
 // Company-wide configuration for apps and services.
 private microsoft.tenant.settings @defaults("isAppAndServicesTrialEnabled isOfficeStoreEnabled") {
-  // Unique synthetic ID based on the parent tenant.
-  id string
   // Controls whether users can start trial subscriptions for apps and services in your organization.
   isAppAndServicesTrialEnabled bool
   // Controls whether users can access the Microsoft Store
   isOfficeStoreEnabled bool
 }
 
+// Company-wide settings for Microsoft Forms
+private microsoft.tenant.formsSettings @defaults("isExternalSendFormEnabled isBingImageSearchEnabled") {
```

Comment:
Can we please stop doing using the same resource name twice, once as field and once as resource. There should be no conflict. I recommend renaming the private resource to `microsoft.tenantFormsSettings`